firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -48,27 +48,16 @@
  */
 public class FirebaseInstallations implements FirebaseInstallationsApi {
 
-  @RefreshAuthTokenOption int refreshAuthTokenOption;
-
-  @Override
-  public void setRefreshAuthTokenOption(@RefreshAuthTokenOption int mode) {
-    refreshAuthTokenOption = mode;
-  }
-
-  @Override
-  @RefreshAuthTokenOption
-  public int getRefreshAuthTokenOption() {
-    return refreshAuthTokenOption;
-  }
-
   private final FirebaseApp firebaseApp;
   private final FirebaseInstallationServiceClient serviceClient;
   private final PersistedFid persistedFid;
   private final ExecutorService executor;
   private final Clock clock;
   private final Utils utils;
 
-  private static final long AUTH_TOKEN_EXPIRATION_BUFFER = 3600L; // 1 hour
+  private static final long AUTH_TOKEN_EXPIRATION_BUFFER_IN_SECS = 3600L; // 1 hour
+
+  private CountDownLatch latch = new CountDownLatch(1);
```

Comment:
I don't think this is a good solution... It's non-final, it means that this class can't be reused, and I don't think it's sufficient to express the state you're trying to express

Reply:
Ack.